### DISCUSSION TITLE
Updating Cookie Info causes Access Denied and logs me out of OnlyFans - did they change something?

### DISCUSSION THREAD

**Discussion by mbiasetti** [UltimaHoarder/UltimaScraper#1791] [Q&A]
Anyone else experience this? I've been running this fine on windows for months. Now, I log in through chrome, use the cookie helper to get my info, and update my auth.json file. As soon as I try to authenticate through the this program I get Access Denied with this error (Something went wrong. [BB91C8D6]), and I simultaneously watch my browser session get logged out. Not even sure where to start with a workaround.

**Comment by fightcancer2** [comment]
Exact same error. Then OF logs me out on the website. More discussion here: https://github.com/DIGITALCRIMINAL/OnlyFans/issues/1780

Names: Username = username  | 0 = All | 1 = default
0
Auth (V1) Attempt 1/10
Something went wrong. [BB91C8D6]
Auth (V1) Attempt 2/10
Access denied.
Auth (V1) Attempt 3/10
Access denied.
Auth (V1) Attempt 4/10
Access denied.
Auth (V1) Attempt 5/10
Access denied.
Auth (V1) Attempt 6/10
Access denied.
Auth (V1) Attempt 7/10
Access denied.
Auth (V1) Attempt 8/10
Access denied.
Auth (V1) Attempt 9/10
Access denied.
Auth (V1) Attempt 10/10
Access denied.
Scraping Paid Content
Scraping Subscriptions
There's nothing to scrape.
Archive Completed in 0.28 Minutes
Pausing scraper for 0 seconds.

**Reply by wooki0** [reply]
same here

**Reply by mbiasetti** [reply]
I realized I was on an old build, so I updated to 7.4.1. Ran the updated requirements, which installed just fine.

The build notes said that auth changed, but this was back in July. I noticed that when I copied my session info from the cookie helper into auth.json, it immediately got modified after I selected OF from the first prompt. 

I'll keep mucking around, but not making much progress.

**Reply by fightcancer2** [reply]
I just got it working.  Does your auth.json look like this?

yyyyyyy = your auth id which is repeated thrice below
xxxxxxxxxxxxxxxxxxxxx = your session ID
zzzzzzzzzzzzzzzzzzzzz = your x-bc header value

{
"auth": {
"username": "default",
"cookie": "auth_id=yyyyyyy; sess=xxxxxxxxxxxxxxxxxx; auth_hash=; auth_uniq_yyyyyyy=; auth_uid_yyyyyyy=;",
"x_bc": "zzzzzzzzzzzzzzz",
"user_agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:95.0) Gecko/20100101 Firefox/95.0",
"email": "",
"password": "",
"hashed": false,
"support_2fa": true,
"active": true
}
}

**Reply by XYZ331** [reply]
I have the same issue. keeps longing me out. I used the code you posted above and plugged in my numbers and still the same thing. how did you fix it exactly? The auth.jason file keeps removing everything to a default state after I run it and get the error.

**Reply by Arcwellington** [reply]
You mention "x_bc": "zzzzzzzzzzzzzzz", in your comment but this does not appear for me. 
Is there some kind of option that needs to be ammended to set this?

**Reply by demib72** [reply]
Might be referring to the latest build (7.6.1) over at https://github.com/DIGITALCRIMINALS/OnlyFans/, I guess the author lost control of this repository.

**Comment by jozipaulo** [comment]
This is an old version of the scraper program. The latest is 7.6.1 at this time (26 Dec 2021)
You can locate the latest version here - https://github.com/DIGITALCRIMINALS/OnlyFans
Looks like this DIGITALCRIMINAL not DIGITALCRIMINALS repository is no longer active.

**Reply by wooki0** [reply]
1 download newest version 7.6.1
2 extract to a directory you want to use for ripping
3 install latest python version and Microsoft visual studio 14
4 once all thats installed then open a power shell window from the onlyfans 7.6.1 directory by holding down shift and right clicking then choosing open power shell window here
5 run this  command: python3.10 -m pip install --upgrade -r requirements.txt 
python version may vary 
6. if command is sucessful then run start.ofd
7. you should then be able to update your config file and save it you can find the info via inspect element from your browser there's more walk throughs on this site. that go in to more detail on this my file uses everything but username and pass and auth hash and seems to work with no problems 
8. once its up to date you should be able to run with no problem

maybe help to somebody
